extensions/wikia/PhalanxII/PhalanxSpecialController.class.php

```diff
@@ -197,10 +197,10 @@ private function blockDataForForm() {
 		// block creation
 		$pager = new PhalanxPager();
 
-		$data['checkBlocker'] = $this->wg->Request->getText( 'wpPhalanxCheckBlocker' , '');
+		$data['checkBlocker'] = $this->wg->Request->getText( 'wpPhalanxCheckBlocker' , '' );
 		$data['checkId'] = $this->wg->Request->getIntOrNull( 'id' );
 		$data['type'] = $pager->getSearchFilter();
-		$data['text'] = $this->wg->Request->getText('target' , ''); // prefill the filter content using target URL parameter
+		$data['text'] = $this->wg->Request->getText( 'target' , '' ); // prefill the filter content using target URL parameter
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)